AdyenTests/Card Tests/Form/FormCardNumberItemTests.swift

```diff
@@ -90,19 +90,19 @@ class FormCardNumberItemTests: XCTestCase {
         
         // Clearing the field should bring back both logos.
         item.value = ""
-        cardTypeProvider.requestCardType(for: item.value, supported: supportedCardTypes) { response in
-            item.didChange(detectedCards: response, for: item.value)
```

Comment:
`FormCardNumberItem ` no longer responsible for handling visual state. If no cards recognised - no card shown.